svm-transaction/Cargo.toml

```diff
@@ -10,4 +10,13 @@ license = { workspace = true }
 edition = { workspace = true }
 
 [dependencies]
-solana-sdk = { workspace = true }
+solana-hash = { workspace = true }
+solana-program = { workspace = true, default-features = false }
```

Comment:
Is there a reason to not use `solana-instruction` and `solana-message` instead? I'm assuming because of `system_instruction`?

Reply:
oh it was just because solana-message wasn't available when i started this. I've removed solana-program now